src/Microsoft.Azure.SignalR.Protocols/ConnectionMessage.cs

```diff
@@ -130,4 +130,25 @@ public ConnectionDataMessage(string connectionId, ReadOnlySequence<byte> payload
         /// </summary>
         public ReadOnlySequence<byte> Payload { get; set; }
     }
+
+    /// <summary>
+    /// A update-connection message.
+    /// </summary>
+    public class UpdateConnectionMessage : ConnectionMessage
```

Comment:
Why is it called UpdateConnection?

Reply:
Want to make it more general so we might add more fields later. Any suggestion?

Reply:
So the intent is that we may add more fields in the future and only the ones that have values marked things that have changed?